wiki/Councils.md

```diff
@@ -10,6 +10,7 @@ For scripts that need postcodes, these should be provided in double quotes and w
 This document is still a work in progress, don't worry if your council isn't listed - it will be soon!
```

Comment:
I ran `make update-wiki`, which seems to have pulled in a few changes. Should this only be run once merged or something?

Reply:
I always thought it ran automatically on merge